src/Controller/ResetPasswordController.php

```diff
@@ -147,10 +140,11 @@ private function processRequestForm(FormInterface $form, Request $request, Maile
         }
 
         $email = (new TemplatedEmail())
+            // @TODO - Set correct email address and name
```

Comment:
Good catch- we should probably ask for these in the `make:` command

Reply:
done